src/openxr/OpenXRApi.cpp

```diff
@@ -2502,16 +2582,22 @@ int OpenXRApi::get_external_texture_for_eye(int eye, bool *has_support) {
 	return 0;
 }
 
-void OpenXRApi::process_openxr() {
-	if (!initialised) {
-		return;
-	}
-
-	XrResult result;
+const char *session_states[] = {
+		"XR_SESSION_STATE_UNKNOWN",
```

Comment:
Hmm, wonder if we should have this at the top of our source, though I doubt we'll be using it in many places.